Cargo.toml

```diff
@@ -2,14 +2,15 @@
 resolver = "2"
 
 members = [
-    "lightning",
-    "lightning-block-sync",
-    "lightning-invoice",
-    "lightning-net-tokio",
-    "lightning-persister",
-    "lightning-background-processor",
-    "lightning-rapid-gossip-sync",
-    "lightning-custom-message",
+	"lightning",
+	"lightning-block-sync",
+	"lightning-invoice",
+	"lightning-net-tokio",
+	"lightning-persister",
+	"lightning-background-processor",
```

Comment:
looks like there is a additional tab here, or it is missing a tab in the `exclude` array?